pkgs/by-name/je/jellyfin-tui/package.nix

```
@@ -0,0 +1,35 @@
+{
+  lib,
+  fetchFromGitHub,
+  rustPlatform,
+  pkg-config,
+  openssl,
+  mpv,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "jellyfin-tui";
+  version = "v1.0.5";
+
+  src = fetchFromGitHub {
+    owner = "dhonus";
+    repo = pname;
+    rev = version;
+    hash = "sha256-dME3oM3k5TGjN8S/93Crt3vw8+KjZWivkVzg+eqwfe4=";
+  };
+
+  cargoHash = "sha256-hogGwkaNDE4yPYgmmG9wfDPlNTUaVqlJzCW31lrX6Ic=";
+
+  meta = {
+    description = "tui music client for jellyfin";
```

```suggestion
    description = "TUI music client for Jellyfin";
```
The message should start with a capital letter, and the acronym should use all capital letters.

Furthermore, would it make sense to use the short description from the upstream repo instead of a custom one?